systemtest/src/main/java/io/strimzi/systemtest/templates/specific/AdminClientTemplates.java

```diff
@@ -25,21 +28,127 @@ public class AdminClientTemplates {
 
     private AdminClientTemplates() {}
 
-    public static Deployment scramShaAdminClient(String namespaceName, String userName, String adminName, String bootstrapName, String additionalConfig) {
-        String finalAdditionalConfig = getAdminClientScramConfig(namespaceName, userName) + "\n" + additionalConfig;
+    ///////////////////////////////////////////
+    //              TlS (SSL)
+    ///////////////////////////////////////////
 
-        return defaultAdminClient(namespaceName, adminName, bootstrapName, finalAdditionalConfig).build();
+    /**
+     * Deploys an AdminClient with TLS enabled, with only essential configuration.
+     *
+     * @param namespaceName The Namespace in which to deploy the AdminClient and where Kafka resides.
+     * @param userName The Kafka userName to correctly configure TLS.
+     * @param adminName The name of the AdminClient deployment.
+     * @param clusterName The name of the Kafka cluster to which the AdminClient will connect.
+     * @param bootstrapName The name of the Kafka bootstrap server to use for the initial connection.
+     */
+    public static Deployment tlsAdminClient(String namespaceName, String userName, String adminName, String clusterName, String bootstrapName) {
+        return tlsAdminClient(namespaceName, userName, adminName, clusterName, bootstrapName, "");
     }
 
-    public static DeploymentBuilder defaultAdminClient(String namespaceName, String adminName, String bootstrapName) {
-        return defaultAdminClient(namespaceName, adminName, bootstrapName, "");
+    /**
+     * Creates a Deployment configuration for an AdminClient with TLS settings.
+     */
+    public static Deployment tlsAdminClient(String namespaceName, String userName, String adminName, String clusterName, String bootstrapName, String additionalConfig) {
+        final List<EnvVar> tlsEnvs = buildTLSUserCredentials(userName);
+        final String finalAdditionalConfig = "sasl.mechanism=GSSAPI\n" + "security.protocol=" + SecurityProtocol.SSL + "\n"  + "\n" + additionalConfig;
+
+        return plainAdminClient(namespaceName, adminName, bootstrapName, finalAdditionalConfig)
```

Comment:
FMPOV it's a bit strange to have method `tlsAdminClient` which returns modified `plainAdminClient`. Maybe it would make sense to have `defaultAdminClient` as we have for other clients?

Reply:
good point thanks.